### DISCUSSION TITLE
How to add a plugin

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#9742] [Help] [answered]
Hello everyone.
How to add a plugin?
@melohagan

**Comment by shogunpurple** [comment, marked as answer]
https://docs.budibase.com/docs/custom-plugin

**Reply by rafikApp** [reply]
![pl](https://user-images.githubusercontent.com/106697208/220079293-957aaa8f-0c34-4c0e-b112-4a7cc6a2a7b5.PNG)


I get this error when I try to add this plugin  https://github.com/FlaminWrap/budibase-signature-component



![plugin](https://user-images.githubusercontent.com/106697208/220079303-1d8147a8-9f30-4cb1-93e1-4d04b50dc3d4.PNG)

**Reply by melohagan** [reply]
Hey @rafikApp 

That repo may not be configured correctly. Change the *Source* to **URL** and use this URL: https://github.com/FlaminWrap/budibase-signature-component/releases/download/1.2.8/Budibase-Signature-Plugin-1.2.8.tar.gz

**Reply by rafikApp** [reply]
It works, Thank you @melohagan
